layouts/shortcodes/kf-latest-version.html

```diff
@@ -1 +1 @@
-v0.6.2
\ No newline at end of file
+v0.7.0-rc.4
```

Comment:
We have rc.5 now - should we update this now or later? We are quite likely to have another rc today or tomorrow.

Reply:
Well spotted. The change to the kf-latest-version file is already merged, with the value for RC5. I'll sync this PR. Then we can update the `kf-latest-version` to RC6 etc as individual PRs. (I put it into all the deployment PRs too, for review convenience.)